### PR TITLE
[gatus] Add gatus chart

### PR DESCRIPTION
**Pull request by funkypenguin**
Hey guys!

Here's another quick chart - this one is modeled after the homer chart, and introduces Gatus, a developer-friendly status page app, which can be entirely configured using a config file (and therefore a configmap).

Any questions, shout out! :)

D

### CODE REVIEW

File: charts/gatus/values.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: twinproduction/gatus
+  # -- image tag
+  tag: v3.8.0
```
Comment (onedr0p) [inline charts/gatus/values.yaml:12]:
```suggestion
  # @default -- chart.appVersion
  tag:
```

File: charts/stable/gatus/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - name: funkypenguin
+    email: davidy@funkypenguin.co.nz
+dependencies:
+  - name: common
+    repository: https://library-charts.k8s-at-home.com
+    version: 4.4.2
+annotations:
+  artifacthub.io/changes: |-
+    - kind: added
+      description: Initial version
```
Comment (onedr0p) [inline charts/stable/gatus/Chart.yaml:22]: Lots of metadata updates need to happen in here.

**Comment by onedr0p** [comment]
You have some cleaning up to do. It seems like you commited files to `./charts and not `./charts/stable`

**Comment by funkypenguin** [comment]
Apologies, that was careless of me. I've fixed all of the above issues, ready for a recheck :)

**Comment by onedr0p** [comment]
Appears there's issues with the chart being able to run. Check the workflow run.

**Pull request merged**